hi-ml-multimodal/README.md

```diff
@@ -1,11 +1,71 @@
-# Multimodal
+# HI-ML Multimodal Toolbox
 
-## Developer setup
+This toolbox provides models for working with multi-modal health data.
+The code is available on [GitHub][1] and [Hugging Face 🤗][6].
 
-Make file commands:
+## Getting started
 
-* `make pip_local` to install the `hi-ml-multimodal` package in editable mode. This must happen before running tests.
-* `make build` to build the package
-* `make mypy` to run `mypy`
-* `make check` to run `flake8`, `mypy` and `pyright`
-* `make clean` to clean up all temporary files and folders
+The best way to get started is by running the [phrase grounding notebook][2].
+All the dependencies will be installed upon execution, so Python 3.7 and [Jupyter][3] are the only requirements to get started.
```

Comment:
If someone has already setup the package and is looking to run some examples, will this overwrite their version of the package? I imagine this would be frustrating for someone trying to test out this notebook on their local branch or if they have the package setup and just want to run a notebook.

Reply:
> If someone has already setup the package and is looking to run some examples, will this overwrite their version of the package?

No, I think `pip` would find an installed `hi-ml-multimodal` package and not overwrite from the notebook.

Reply:
Even if we specifically ask it to install from `main` as we do in the notebook? I would expect that it would find that the version installed locally is a different version and uninstall it. Trying it locally this is what I see on my machine.

Reply:
Ok I see. I think it's still better to leave it as is. We can mention within the notebook that the first cell shouldn't be run if the user wants to use the local environment. Developers of HI-ML will be 1) way rarer users of the notebook and 2) savvy enough to understand that the package from git should be used. The most likely scenario, I think, is someone running the notebook on Binder or simply to play around with it, not necessarily to work on the library.